tests/pipeline/test_blaze.py

```diff
@@ -1185,32 +1183,18 @@ def test_id_take_last_in_group_macro(self):
         fields['other'] = fields['value']
 
         expected = pd.DataFrame(
-            columns=[
-                'other', 'value',
-            ],
-            data=[
-                [np.nan,      1],  # 2014-01-01 Equity(65 [A])
-                [np.nan,      1],             # Equity(66 [B])
-                [np.nan,      1],             # Equity(67 [C])
-                [1,           2],  # 2014-01-02 Equity(65 [A])
-                [1,           2],             # Equity(66 [B])
-                [1,           2],             # Equity(67 [C])
-                [2,           2],  # 2014-01-03 Equity(65 [A])
-                [2,           2],             # Equity(66 [B])
-                [2,           2],             # Equity(67 [C])
-            ],
-            index=pd.MultiIndex.from_product(
-                (self.dates, self.asset_finder.retrieve_all(
-                    self.ASSET_FINDER_EQUITY_SIDS
-                )),
-            ),
+            data=[[np.nan, 1],   # 2014-01-01
+                  [1,      2],   # 2014-01-02
+                  [2,      2]],  # 2014-01-03
+            columns=['other', 'value'],
+            index=self.dates,
         )
-        self._test_id(
+        self._test_id_macro(
             df,
             var * Record(fields),
             expected,
             self.asset_finder,
-            ('value', 'other'),
+            ('other', 'value'),
```

Comment:
This order had to align with the order of the columns of `expected`, as this order matters when adding these columns as factor inputs in `_test_id_macro` (previously I don't think order mattered when just doing `pipeline.add`).